logstash-core/lib/logstash/modules/logstash_config.rb

```diff
@@ -84,6 +84,10 @@ def elasticsearch_output_config(type_string = nil)
 CONF
   end
 
+  def password_value(password)
+    password.is_a?(LogStash::Util::Password) ? password.value : password
```

Comment:
@robbavey Why isn't this always a `LogStash::Util::Password`?

Reply:
@suyograo I think it probably is - it was more unfamiliarity with all the code that made me slightly wary. I can change it though